### PR TITLE
updated viewbook images and scroll click on gold cards

### PR DESCRIPTION
**Pull request by keeeeeegan**
Updates requested from Patrick:

* Two of the viewbook images needed updating to remove a black border on one or more edges of the image.
* Clicking on the gold cards was scrolling the page to the appropriate heading, but wasn't scrolling far enough so the headings were being cut off.

### CODE REVIEW

**Comment by rjucf** [comment]
not a big deal, but caching $('#sections-menu') with a variable would be better.
var $sectionsMenu = $('#sections-menu');

**Comment by keeeeeegan** [comment]
Added an update to cache the sections-menu.

File: dev/admissions-page/admissions.js [hunk 1]
```diff
@@ -25,10 +25,17 @@ var enableScrollSpy = function($) {
...
     $target = $target.length ? $target : $('[name=' + this.hash.slice() + ']');
 
-    var scrollTo = $target.offset().top - 50;
+    var scrollTo = $target.offset().top;
+
+    if ($sectionsMenu.length > 0) {
+      scrollTo = scrollTo - $sectionsMenu.height() + 20;
+    }
+
+    var scrollTo = scrollTo - 150;
```
Comment (cjg89) [inline dev/admissions-page/admissions.js]: `var` declaration should be removed here (scrollTo is already defined on line 32).

File: dev/admissions-page/admissions.js [hunk 2]
```diff
@@ -25,10 +25,17 @@ var enableScrollSpy = function($) {
...
 
+    var $sectionsMenu = $('#sections-menu');
     var $target = $(this.hash);
     $target = $target.length ? $target : $('[name=' + this.hash.slice() + ']');
 
-    var scrollTo = $target.offset().top - 50;
+    var scrollTo = $target.offset().top;
+
+    if ($sectionsMenu.length > 0) {
+      scrollTo = scrollTo - $sectionsMenu.height() + 20;
```
Comment (cjg89) [inline dev/admissions-page/admissions.js]: Adding some sort of grouping may be helpful here (e.g. `scrollTo - ($sectionsMenu.height() + 20)` or `(scrollTo - $sectionsMenu.height()) + 20`), depending on what you're trying to accomplish.

**Comment by cjg89** [comment]
You may also want to redact the QA link... we probably don't want links to our QA instances getting crawled by bots.

**Comment by cjg89** [comment]
Can this be closed?

**Comment by cjg89** [comment]
This got closed by accident when the rc branch was renamed.  It doesn't look like the branch was ever merged in; if the functionality is still needed, please merge in to rc-v2.1.0.  Thanks!

**Pull request closed without merging**